src/core/dense_set.cc

```diff
@@ -648,6 +648,8 @@ void DenseSet::Delete(DensePtr* prev, DensePtr* ptr) {
 
       DenseLinkKey* plink = prev->AsLink();
       DensePtr tmp = DensePtr::From(plink);
+      // Transfer TTL flag
```

Comment:
I think we need to set the ttl bit in the wrapping pointer. See comments in AddOrReplaceObj

Reply:
Wrapping is destroyed so we need to set TTL of object. This is same behavior as what `Grow()` does.